files/__addonLocation__/package.json

```diff
@@ -10,18 +10,25 @@
   "author": "",
   "files": [
     "addon-main.cjs",
-    "dist"
+    "dist",
+    "declarations"
   ],
-  "scripts": {
-    "build": "rollup --config",
+  "scripts": {<% if (typescript) { %>
+    "build": "concurrently 'npm:build:*'",
+    "build:js": "rollup --config",
+    "build:types": "glint --build", <% } else { %>
+    "build": "rollup --config", <% } %>
     "lint": "concurrently 'npm:lint:*(!fix)' --names 'lint:'",
     "lint:fix": "concurrently 'npm:lint:*:fix' --names 'fix:'",
     "lint:hbs": "ember-template-lint . --no-error-on-unmatched-pattern",
     "lint:js": "eslint . --cache",
     "lint:hbs:fix": "ember-template-lint . --fix --no-error-on-unmatched-pattern",
     "lint:js:fix": "eslint . --fix",<% if (typescript) { %>
-    "lint:types": "glint",<% } %>
-    "start": "rollup --config --watch",
+    "lint:types": "glint",
+    "start": "concurrently 'npm:start:*'",
+    "start:js": "rollup --config --watch --no-watch.clearScreen",
```

Comment:
I didn't encounter the need for `--no-watch.clearScreen` and would recommend keeping commands simple.

```suggestion
    "start:js": "rollup --config --watch",
```

The above matches the command for the JavaScript case.

Reply:
Without this, both rollup and glint (tsc) fight for clearing the screen.
To prevent accidentally hiding issues, we need the screen to not clear on (re)build (which is what the flag does).

I have an issue for myself to investigate for the glint side of this: https://github.com/typed-ember/glint/issues/597#issuecomment-1620272106

Reply:
Sounds good, thanks.